content/en/developers/integrations/log_pipeline.md

```diff
@@ -16,111 +16,159 @@ further_reading:
   tag: Documentation
   text: Learn about log pipelines
 title: Create a Log Pipeline
-description: Learn how to create a Datadog Log integration.
+description: Learn how to create a Datadog Log Integration.
 ---
 ## Overview
 
-This page walks Technology Partners through creating a log pipeline. A log pipeline is required if your integration is sending in logs. 
+This page walks Technology Partners through creating a log pipeline. A log pipeline is required if your integration sends logs to Datadog.
 
-## Log integrations
+## Integration prerequisites
 
-Use the [Logs Ingestion HTTP endpoint][1] to send logs to Datadog.
-
-## Development process
+In developing your integration to send logs to Datadog follow these guidlines to ensure the best experience for your users.
 
 ### Guidelines
 
-When creating a log pipeline, consider the following best practices:
-
-Map your data to Datadog's standard attributes
-: Centralizing logs from various technologies and applications can generate tens or hundreds of different attributes in a Log Management environment. Integrations must rely as much as possible on the [standard naming convention][17].
+Before creating a log pipeline, consider the following guidlines and practices:
 
-Set the `source` tag to the integration name.
-: Datadog recommends that the `source` tag is set to `<integration_name>` and that the `service` tag is set to the name of the service that produces the telemetry. For example, the `service` tag can be used to differentiate logs by product line. </br></br> For cases where there aren't different services, set `service` to the same value as `source`. The `source` and `service` tags must be non-editable by the user because the tags are used to enable integration pipelines and dashboards. The tags can be set in the payload or through the query parameter, for example, `?ddsource=example&service=example`. </br></br> The `source` and `service` tags must be in lowercase. 
```

Comment:
We should keep this section in the doc. Questions have come up in the past on source & service as well as ways to send the tags to DD.

Reply:
Hey @emarsha94 I actually have kept this but it appears lower in the text.
The problem I have is originally the doc refers to "Datadog Integrations" then Log Pipelines, so the concepts of Service, Integration Name and Source are confused with the actual Integration. Rather, I put the information about `source` and `service` at lines 85 onwards.

Reply:
Done - added this in a new commit.